README.md

```diff
@@ -2,3 +2,24 @@
 
 [![Build Status](https://travis-ci.org/dremio-hub/dremio-hbase-connector.svg?branch=master)](https://travis-ci.org/dremio-hub/dremio-hbase-connector)
 
+## Usage Instructions
+
+* From the project root run `mvn clean install`. The hbase connector jar should now be present
+in the `target` directory.
+
+* Copy the connector jar over to the `<DREMIO_HOME>/jars` folder on all Dremio nodes.
+
+* Stop Dremio.
```

Comment:
Consider giving the user the command that does this, as we did in step 1 with the maven command.

Reply:
Done